src/core_plugins/kibana/public/visualize/embeddable/visualize_embeddable.js

```diff
@@ -0,0 +1,115 @@
+import { PersistedState } from 'ui/persisted_state';
+import { Embeddable } from 'ui/embeddable';
+import chrome from 'ui/chrome';
+import _ from 'lodash';
+
+export class VisualizeEmbeddable extends Embeddable  {
+  constructor({ onEmbeddableStateChanged, savedVisualization, editUrl, loader }) {
+    super();
+    this._onEmbeddableStateChanged = onEmbeddableStateChanged;
+    this.savedVisualization = savedVisualization;
+    this.loader = loader;
+
+    const parsedUiState = savedVisualization.uiStateJSON ? JSON.parse(savedVisualization.uiStateJSON) : {};
+    this.uiState = new PersistedState({
+      ...parsedUiState,
+    });
```

Comment:
Hmm, yea I'm not sure if there wa s areason for this. I'll switch it and see how tests behave.  If there is a reason for it, I should have written a comment since it's not obvious.